packages/SwingSet/package.json

```diff
@@ -100,6 +100,6 @@
     "access": "public"
   },
   "typeCoverage": {
-    "atLeast": 76.28
+    "atLeast": 76.25
```

Comment:
What should I look for when reviewing changes in typeCoverage floor?

Reply:
Ideally it goes up. It should never go down _too much_, per reviewer's judgement